.github/workflows/manual-release.yml

```diff
@@ -0,0 +1,44 @@
+name: manual-release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    # this job should only ever be run on master
+    if: github.ref_type == 'branch' && github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    # minimal permissions for Get Artifact Publish Token
+    permissions:
+      id-token: write
+    steps:
+      - uses: actions/checkout@v3
+      # Make sure that master and the latest git tag are pointing at the same commit
+      # This will go some way to ensuring that we are building the latest version
+      # and that a git tag was applied to master.
+      - name: Ensure master and latest tag are the same commit
+        run: |
+          TAG=$(git describe --tag --abbrev=0)
+          [ "$(git rev-parse master)" != "$(git rev-parse $tag)" ] && echo "master and $tag are different commits" && exit 1
+      # Get the token we need to publish to packages.atlassian.com
+      - name: Get publish token
+        id: publish-token
+        uses: atlassian-labs/artifact-publish-token@v1.0.1
+        with:
+          output-modes: environment
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@master
+        with:
+          platforms: 'amd64,arm64'
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@master
+      # - name: Release
+      #   run: |
+      #     echo "Executing release on tag build $GITHUB_REF_NAME"
+      #     ARCH=linux CPU_ARCH=amd64 make release-ci
+      #     ARCH=linux CPU_ARCH=arm64 make release-ci
+      # - name: Push Docker Manifest
+      #   run: |
+      #     echo "Executing release-manifest on tag build $GITHUB_REF_NAME"
+      #     make release-manifest-ci
```

Comment:
Are these still required or could that be nuked?

Reply:
I believe we still want them, it's just commented out for testing purposes, then it'll be enabled if the builds all work.

Reply:
yes, exactly that.